README.md

```diff
@@ -57,36 +148,45 @@ it('can create a user', function () {
 DOM structure altered via interaction with component:
 
 ```javascript
-it('sets disabled property', function () {
-  this.render(hbs`
-    {{frost-password
-      disabled=true
-    }}
- `)
-
-  expect(
-    this.$('.frost-password').find('input').prop('disabled')
-  ).to.eql(true)
+describe('when disabled property is set', function () {
+  beforeEach(function () {
+    this.render(hbs`
+      {{frost-password
+        disabled=true
+      }}
+    `)
+  })
+
+  it('should set the "disabled" prop on the inner <input> element', function () {
+    expect(this.$('.frost-password input')).to.have.prop('disabled', true)
+  })
 })
 ```
 
 Validate interacting with component fires closure action:
 
 ```javascript
-it('calls onClick closure action', function () {
-  const externalActionSpy = sandbox.spy()
-
-  this.on('externalAction', externalActionSpy)
-
-  this.render(hbs`
-    {{frost-link 'title'
-      onClick=(action 'externalAction')
-    }}
-  `)
+describe('when onClick property is set', function() {
+  let clickHandler
+  beforeEach(function() {
+    clickHandler = sinon.stub()
```

Comment:
Curious, why? I've always used stubs, b/c I never want the original method to be called.